pkg/systemd/host.js

```diff
@@ -296,20 +292,10 @@ PageServer.prototype = {
                     model.SubStatus = timesyncd_status;
             }
 
-            var popover_html = Mustache.render(self.ntp_status_tmpl, model);
-            if (popover_html != $elt.attr('data-content')) {
-                $elt.attr("data-content", popover_html);
-                // Refresh the popover if it is open
-                if ($elt.data('bs.popover').tip().hasClass('in'))
-                    $elt.popover('show');
-            }
-
-            var icon_html = Mustache.render(self.ntp_status_icon_tmpl, model);
-            $elt.html(icon_html);
```

Comment:
I think we should leave the spinner and error icons.

Reply:
I agree.